Flutter/calendar/appointments.md

```diff
@@ -1013,7 +1013,7 @@ Gets an occurrence at the specified date within a series of recurring appointmen
 Gets the pattern appointment for the specified occurrence by using the [getPatternAppointment](https://pub.dev/documentation/syncfusion_flutter_calendar/latest/calendar/CalendarDataSource/getPatternAppointment.html).
 
 {% tabs %}
-{% highlight dart hl_lines="12 14 15" %}
+{% highlight dart hl_lines="12 13 14 15" %}
```

Comment:
https://help.syncfusion.com/flutter/calendar/appointments#get-visible-appointments

Highlight wrongly

Reply:
allready highlighted properly in the Get visible appointments topic.